src/main/java/com/osiris/autoplug/client/configs/SSHConfig.java

```diff
@@ -0,0 +1,124 @@
+import com.osiris.dyml.Yaml;
+import com.osiris.dyml.YamlSection;
+import com.osiris.dyml.exceptions.*;
+import com.osiris.jlib.logger.AL;
+import com.osiris.autoplug.client.Main;
+
+import java.io.IOException;
+
+public class SSHConfig extends MyYaml {
+    
+    public YamlSection enabled;
+    public YamlSection port;
+    public YamlSection auth_method;
+    public YamlSection allowed_keys_path;
+    public YamlSection server_private_key;
+    public YamlSection username;
+    public YamlSection password;
+
+    public SSHConfig() throws IOException, DuplicateKeyException, YamlReaderException, IllegalListException, NotLoadedException, IllegalKeyException, YamlWriterException {
+        super(System.getProperty("user.dir") + "/autoplug/ssh.yml");
+
+        addSingletonConfigFileEventListener(e -> {
+            try {
+                Main.sshServerSetup.stop();
+                Main.sshThread.join();
+            } catch (Exception e1) {
+                AL.error("Failed to stop SSH Server!", e1);
+            }
+            if (enabled.asBoolean()) {
+                Main.sshServerSetup = new SSHServerSetup();
+                Main.sshThread = new Thread(() -> {
+                    try {
+                        Main.sshServerSetup.start();
+                    } catch (Exception e1) {
+                        AL.error("Failed to start SSH Server!", e1);
+                    }
+                });
+                Main.sshThread.start();
+            }
+        });
+        
+
+        lockFile();
+        load();
+        String name = getFileNameWithoutExt();
+        put(name).setComments(
+                "#######################################################################################################################",
+                "    ___       __       ___  __",
+                "   / _ |__ __/ /____  / _ \\/ /_ _____ _",
+                "  / __ / // / __/ _ \\/ ___/ / // / _ `/",
+                " /_/ |_\\_,_/\\__/\\___/_/  /_/\\_,_/\\_, /",
+                "                                /___/ SSH-Config",
+                "Thank you for using AutoPlug!",
+                "You can find detailed installation instructions here: https://autoplug.one/installer",
+                "If there are any questions or you just want chat, join our Discord: https://discord.gg/GGNmtCC",
+                " ",
+                "#######################################################################################################################",
+                "Note: Changes to this file probably require you to enter '.con reload' to have affect.");
+        
+        enabled = put(name, "enabled").setDefValues("false")
+            .setComments(
+                "Enables a SSH console for remote access to AutoPlug.",
+                "You can connect to it via any SSH client like Putty, a custom script, or the built-in SSH client in Windows, Linux distros, MacOS, etc.");
```

Comment:
Add:
Uses apache sshd 2.13.0 internally, make sure it doesn't contain vulnerabilities before using here: https://mvnrepository.com/artifact/org.apache.sshd/sshd-core

Reply:
Checked, none listed. Add what?

Reply:
I meant add it as a comment.